src/components/top-bar/styles.ts

```diff
@@ -0,0 +1,69 @@
+import { styled } from '@linaria/react'
+import { isDesktop, isDesktopOrBelow, isTablet, isTabletOrBelow } from '../../styles/media'
+import { ElButtonGroup } from '../button-group'
+import { css } from '@linaria/core'
+
+export const ElTopBarProfile = styled.div`
+  padding: var(--spacing-1, 4px) var(--spacing-none, 0px);
+`
+
+export const ElTopBarMainNav = styled(ElButtonGroup)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: var(--spacing-6, 24px);
+  flex-grow: 1;
+`
+
+export const ElTopBarSecondaryNav = styled(ElButtonGroup)`
+  flex-wrap: nowrap;
+  padding-right: var(--spacing-spacing-2, 8px);
+`
+
+export const ElTopBar = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: var(--size-10, 56px);
+  padding: var(--spacing-2, 8px) var(--spacing-5, 20px);
+  border-bottom: var(--border-default, 1px) solid var(--outline-default, #e5e9ed);
+  background: var(--fill-white, #fff);
+
+  ${isDesktopOrBelow} {
+    padding: var(--spacing-2, 8px) var(--spacing-4, 16px);
+
+    ${ElTopBarMainNav}, ${ElTopBarSecondaryNav} {
+      display: none;
+    }
+  }
+
+  ${isTabletOrBelow} {
+    ${ElTopBarProfile} {
+      display: none;
+    }
+  }
+`
+
+export const ElTopBarSearch = styled.div`
+  padding-right: var(--spacing-spacing-2, 8px);
+
+  ${isTablet} {
+    width: 216px;
+    padding: var(--spacing-1, 4px) var(--spacing-4, 16px) var(--spacing-1, 4px) var(--spacing-none, 0px);
+  }
+`
+
+export const ElTopBarMobileNav = styled.div`
+  display: inline-block;
+  padding-right: var(--spacing-spacing-2, 8px);
+
+  ${isDesktop} {
+    display: none;
+  }
+`
+
+export const elTopBarMenuPopover = css`
+  // To adjust the menu popover's Y offset so it appears below the top bar instead of directly below the button.
+  // The "important" rule is used to override the Menu's auto-anchor yOffset.
+  top: 44px !important;
```

Comment:
Unfortunately, there is no CSS variable that fits this requirement. The closest option is probably `spacing-8`, which is 2.5rem, but 44px is equal to 2.75rem. Since this is likely a specific case and the top bar height is fixed, I believe we can proceed with this approach.

Reply:
**note:** This is fine for now, but I think we could improve on how we facilitate the unique offset in future; probably when we upgrade to React 19 and get access to the HTML popover API we've mentioned previously.